demos/chatroom/experiment.py

```diff
@@ -1,6 +1,10 @@
 """Coordination chatroom game."""
 
 import dallinger as dlgr
+try:
+    unicode = unicode
+except NameError:  # Python 3
+    unicode = str
 
```

Comment:
Now that we've got this in 2 places, it'd be better to import it from a `compat` module.